CONTRIBUTING.md

```diff
@@ -9,6 +9,21 @@ Please raise any bug reports on the Kabanero project repository's GitHub issue t
 
 A good bug report is one that make it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
 
+Please open issues in their appropriate repository: 
+
+| Repository | Descritpion |
+| --- | --- |
+| [Appsody Extenstion](https://github.com/kabanero-io/appsodyExtension) | This repository is an extension to Codewind that adds support for Appsody projects. |
+| [Blogs](https://github.com/kabanero-io/blogs) | [Blogs](https://kabanero.io/blog) for the Kabanero.io site |
+| [Codewind Templates](https://github.com/kabanero-io/codewind-templates) | Central repository for storing list of supported code templates for use in codewind |
+| [Collections](https://github.com/kabanero-io/collections) | Repo contains source of usable Kabanero Collections |
+| [Docs](https://github.com/kabanero-io/docs) | Common Kabanero documentation and example scripts |
+| [Kabanero CLI](https://github.com/kabanero-io/kabanero-command-line) | Repo for Kabanero-mgmt CLI |
```

Comment:
A command line interface that is used to manage the collections the Kabanero environment presents